specification/app/resource-manager/Microsoft.App/preview/2023-08-01-preview/ManagedEnvironments.json

```diff
@@ -1407,6 +1407,10 @@
             "update"
           ],
           "x-ms-secret": true
+        },
+        "dynamicJsonColumns": {
+          "description": "Boolean indicating whether to parse json string log into dynamic json columns",
```

Comment:
consider making it an extensible enum 

property name - jsonLogParsingConfiguration

with values
DynamicJsonColumns

This allows for future extensibility if u want to expose a different method to transform the output later

Reply:
@rkmanda, This is to indicate whether to use some fluent bit functionality to lift fields in json format to separate columns in LA. It should be ok to be just a true or false. we cannot see other extensibility on this feature. If there is, it should be some different functionality. then a different property should be better.